src/main/java/driver/MainTest.java

```diff
@@ -1,15 +1,35 @@
 package driver;
 
-import fworks.views.SolutionFrame;
-import fworks.views.TestFrame;
-import fworks.views.WelcomeDialog;
+import entities.*;
+import fworks.views.*;
+import ia.controllers.*;
+import ia.presenters.*;
+import uc.user.login.*;
+import uc.user.register.*;
 
 /**
  * A driver for testing the UI
  */
 public class MainTest {
     public static void main(String[] args) {
-        new WelcomeDialog(); // Register/Log-in window
+        // TODO: is it okay to construct things here?
+
+        StateTracker stateTracker = new StateTracker();
+        UserFactory userFactory = new UserFactory(); // TODO: no need to construct if static
```

Comment:
Future PR will make factory methods non-static so that they are constructed in driver and injected into use case interactors